src/org/zaproxy/zap/extension/jruby/ZapAddOn.xml

```diff
@@ -1,6 +1,6 @@
 <zapaddon>
 	<name>Ruby scripting</name>
-	<version>5</version>
+	<version>6</version>
```

Comment:
- The version should still be 5.
 - The changes should mention the update (and in the other ZapAddOn,xml file, for the other the old changes removed).

Reply:
Well, you guys said that the version should be updated in that email thread. ZAP also refused to load the addons because they had the version as the installed instances.

Reply:
What should i do then?

Reply:
That was for Jython, which should have a new version. JRuby version 5 was not yet released to marketplace, so should be kept in 5.

Regarding the installation, you need to uninstall the "older" version or just replace the file.

Reply:
Ok, jruby version rolled back to 5.